CONTRIBUTING.md

```diff
@@ -94,6 +96,28 @@ Afterwards, to clean up:
     2. Remove the modifications to your tsconfig.json
 2. In the directory of _stencil core_, run `npm unlink`
 
+> [!NOTE]  
+> Alternatively you can reference Stencil from a local directory in the `package.json`, e.g.
```

Comment:
I _think_ we'd still need the changes here to the `tsconfig.json` here. It might be worth rewording this so that it doesn't seem like this doesn't require changing `tsconfig.json` - unless perhaps it does an I missed something here?

```suggestion
> Instead of linking, you can reference Stencil from a local directory in the `package.json` after updating your project's `tsconfig.json` file, e.g.
```

Reply:
Totally makes sense, thanks!